index.rst

```diff
@@ -2,6 +2,17 @@
 OpenMEEG Home
 =============
 
+.. raw:: html
+    
+  <div class="leadfield" style="float: right; margin: -7px 0 -10px 0;">
+
+.. image:: _static/leadfield.png
+   :width: 500px
+
+.. raw:: html
+
+  </div>
+
 What is OpenMEEG?
```

Comment:
I have added this leadfield image, I think it is good to directly show what are the output and meshes of OpenMEEG